pythainlp/util/thai_lunar_date.py

```diff
@@ -0,0 +1,828 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: 2016-2024 PyThaiNLP Project
+# SPDX-License-Identifier: Apache-2.0
+"""
+This file is port from
+> https://gist.github.com/touchiep/99f4f5bb349d6b983ef78697630ab78e
+"""
+from datetime import date, timedelta
+from typing import Tuple, Union
+
+
+def XLMod(a: float, b: float) -> int:
+    return a - b * int(a / b)
```

Comment:
This is a modulo function.

In Python, we can just use `a % b`.

The original VBA code needs this function because they VBA Mod doesn't support float.
But as Python % supports both int and float. There's no need to have this function.

Reply:
Done